contrib/metacontroller/UPDGRADE.md

```diff
@@ -0,0 +1,11 @@
+# Upgrade Metacontroller
+
+Metacontroller is pulled from [Kubeflow Pipelines third-party folder](https://github.com/kubeflow/pipelines/tree/master/manifests/kustomize/third-party/metacontroller). To update this component specify the desired KFP version and run the following command in console from the root directory **/**:
+
+```bash
+export KFP_VERSION=2.0.0-alpha.3   # specify KFP version
+kpt pkg update ./contrib/metacontroller@${KFP_VERSION}
```

Comment:
Question: Can we use `Kubeflow version` instead of `KFP version`?

Reply:
I checked with @annajung . It seems that `manifests` repo does not store KFP versions. Instead, one need to set environment variable `COMMIT=<...>` before running [hack/sync-pipelines-manifests.sh](https://github.com/kubeflow/manifests/blob/master/hack/sync-pipelines-manifests.sh).